src/test/resources/config.yaml

```diff
@@ -0,0 +1,5 @@
+
+platformName: ANDROID
+currentSchemes:
+  - scheme1
+  - scheme2
```

Comment:
Do we care whether files end in a new line? I think it would be nice just to get rid of these github warnings but also https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Reply:
Generally no, but I can fix this instance.